src/components/TradeWidget/TokenRow.tsx

```diff
@@ -305,36 +307,42 @@ const TokenRow: React.FC<Props> = ({
       <div>
         <strong>{selectLabel}</strong>
         <span>
-          <button>+ Deposit</button>
+          <TooltipWrapper as="button" tooltip="Deposit">
+            + Deposit
+          </TooltipWrapper>
+          {/* <button>+ Deposit</button> */}
```

Comment:
dead code?

Reply:
Not dead code. Didn't delete as `<TooltipWrapper>` is just an example. Maybe we don't need a tooltip for that button